oblt-cli/deploy-my-kibana/README.md

```diff
@@ -0,0 +1,51 @@
+# <!--name-->oblt-cli/deploy-my-kibana<!--/name-->
+
+[![usages](https://img.shields.io/badge/usages-white?logo=githubactions&logoColor=blue)](https://github.com/search?q=elastic%2Foblt-actions%2Foblt-cli%2Fdeploy-my-kibana+%28path%3A.github%2Fworkflows+OR+path%3A**%2Faction.yml+OR+path%3A**%2Faction.yaml%29&type=code)
+[![test-oblt-cli-deploy-my-kibana](https://github.com/elastic/oblt-actions/actions/workflows/test-oblt-cli-deploy-my-kibana.yml/badge.svg?branch=main)](https://github.com/elastic/oblt-actions/actions/workflows/test-oblt-cli-deploy-my-kibana.yml)
+
+
+<!--description-->
+Run the deploy my Kibana PR.
+<!--/description-->
+
+## Inputs
+<!--inputs-->
+| Name                     | Description                                                 | Required | Default                                 |
+|--------------------------|-------------------------------------------------------------|----------|-----------------------------------------|
+| `comment-url`            | The GitHub Comment URL                                      | `false`  | `${{ github.event.comment.html_url }}`  |
+| `comment-id`             | The GitHub Comment ID                                       | `false`  | `${{ github.event.comment.id }}`        |
+| `issue-url`              | The GitHub Issue URL                                        | `false`  | `${{ github.event.comment.issue_url }}` |
+| `repository`             | The GitHub repository                                       | `false`  | `${{ github.repository }}`              |
+| `user`                   | The GitHub user that triggered the workflow                 | `false`  | `${{ github.triggering_actor }}`        |
```

Comment:
I just took a look at https://github.com/elastic/oblt-actions/tree/main/github/is-member-of

and I feel like we have an inconsistency with naming inputs.

This is not blocking, but we should think about, when to prefix with `github-` or not.

maybe we can revisit this when we eventually need to release v2